Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:2.7
+ADD . /code
+WORKDIR /code
+RUN pip install -r requirements.txt
```

Comment:
Dependencies install should move as up as possible before any other code copying. In this way dependencies which don't change much will be precached and reusable on next image build. I.e:
...
ADD ./requirements.txt
ADD ./requirements_prod.txt
RUN ...
...
//do other commands which will produce different results more often

Reply:
Good idea, done.